docs/get-started.md

```diff
@@ -0,0 +1,91 @@
+# Instrument a Go application or services automatically
+
+You can instrument a Go executable using OpenTelemetry without having
+to write additional code. All you need to do is configure a few environment
+variables and run the instrumentation with elevated privileges.
+
+The following example shows how to instrument a simple Go application
+automatically on a Linux host, through Docker, and using Kubernetes.
+
+## Prerequisites
+
+To instrument an application automatically, you need the following:
+
+- Linux with kernel version 4.19 or higher
+- x64 or ARM processor
+- Docker image or compiled binary of OpenTelemetry Go instrumentation
+
+To compile the instrumentation binary, use Go 1.18 or higher.
+
+## Instrument an application on the same host
+
+To instrument an application on the same host, follow these steps:
+
+1. Set the following environment variables:
+
+  - `OTEL_GO_AUTO_TARGET_EXE`: Full path of the executable you want to
+  instrument. For example, `/home/bin/service_executable`
+  - `OTEL_SERVICE_NAME`: Name of your service or application
+  - `OTEL_EXPORTER_OTLP_ENDPOINT`: Your observability backend. For example,
+  `http://localhost:4317. If you're sending data to the OpenTelemetry Collector
+  over HTTPS, set TLS settings in the OTLP exporter. See the
+  [OTLP Receiver documentation](https://github.com/open-telemetry/opentelemetry-collector/blob/main/receiver/otlpreceiver/README.md)
+  for instructions
```

Comment:
These env vars are only needed for the OpenTelemetry Go Automatic Instrumentation process. I suggest to move this to the "Run the OpenTelemetry Go instrumentation with root privileges." step. E.g.

```
Run the OpenTelemetry Go Automatic Instrumentation with root privileges and following environment variables:
```